### ISSUE TITLE
Email Notifications not working when condition is on an element field entries

### ISSUE THREAD

**Issue by ReedBe** [verbb/formie#330]
**Description**
Email notifications are not working if the send condition is defined on an "element field entries".

**Steps to reproduce**
1. Create a new form
2. Add an "Element field" > "Entries"
3. Add an "Email notifications" with a condition set to the field added in step 2 (with IS condition)

### Form settings
- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.16.1
- Craft version: 3.5.17.1
- Multi-site: Yes

**Additional context**
If the condition type "IS" et remplaced by "CONTAINS" an error is thrown.

**Comment by engram-design** [comment]
Should be fixed in the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.16.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Should be fixed in [1.3.17](https://github.com/verbb/formie/releases/tag/1.3.17)
